### PR TITLE
AZ-386: generate balances for bench accounts

### PR DESCRIPTION
**Pull request by kostekIV**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -359,6 +376,19 @@ def prepare_vesting(chainspec):
     rtm['vesting']['vesting'] = vesting
 
 
+def prepare_benchmark_accounts(chainspec, n_of_accounts, azero_amount):
+    bench_accounts = generate_accounts_from_seeds((str(i) for i in range(n_of_accounts)))
+    balances = []
+    azero = 1000000000000
```
Comment (mike1729) [inline utils.py:382]: replace with `1e12`, is less error prone.

**Review by mike1729 — CHANGES_REQUESTED** [review]

File: utils.py (hunk 1 above)
Comment (kostekIV) [reply utils.py]: extracted `azero` to function that returns 1e12 (it was used in couple of places)

**Review by mike1729 — APPROVED** [review]

**Review by fixxxedpoint — APPROVED** [review]
lgtm

**Review by fixxxedpoint — CHANGES_REQUESTED** [review]
Unfortunately it didn't pass tests. Example invocation: 
```
setup_nodes(n_parties=4, chain='testnet', regions=['eu-central-1'], instance_type='t3.medium', volume_size=8, tag='zbyszko.headshot', benchmark_config={'n_of_accounts': 600000, 'azero_amount': 1}); 
tr(task='dispatch', region_name='eu-central-1', tag='zbyszko.headshot')
``` 
Stuck at: `generating keys & addresses files`

**Review by fixxxedpoint — APPROVED** [review]
lgtm. Perhaphs it would be good ide to give a short comment about that phrase, but up to you.

**Pull request merged**